test/without-slow-down.test.js

```diff
@@ -0,0 +1,36 @@
+import t from 'tap'
+import Fastify from 'fastify'
+import slowDownPlugin from '../index.js'
+import { sleep } from '../lib/helpers.js'
+import { HEADERS } from '../lib/constants.js'
+
+t.test('Should works without delay after 1 second', async t => {
```

Comment:
if you want to have tests both with fake time and with real time, maybe we should call one suite unit and the other integration. clearly we don't want to have too many integration tests as they will be slow

Reply:
I changed the `timeWindow` to 30 seconds and I remove this test with a real timer because it took a lot of time to run it.